pkg/bpf/bpf_linux.go

```diff
@@ -113,7 +115,29 @@ func UpdateElementFromPointers(fd int, structPtr unsafe.Pointer, sizeOfStruct ui
 	}
 
 	if ret != 0 || err != 0 {
-		return fmt.Errorf("Unable to update element for map with file descriptor %d: %s", fd, err)
+
+		var errMsg error
+
+		switch err {
+		case unix.E2BIG:
+			if strings.Contains(mapName, "lb") {
+				errMsg = errors.New("the bpf LB map is full, please consider resizing it with the flag \"bpf-lb-max-entries\". THIS WILL INTERFERE WITH ALL EXISTING CONNECTIONS")
```

Comment:
Since we have the name of the map now, I think it makes sense to add it to the error. (Same goes for all messages below.)